client/src/components/BadReview/BadReviewControllers.tsx

```diff
@@ -38,6 +38,12 @@ export function BadReviewControllers({ courseTasks, courseId }: IBadReviewContro
     }
   }, [taskId, checkType, checkService, courseId]);
 
+  const downloadSolutionsUrls = async () => {
```

Comment:
I guess `async` not needed

Reply:
Yeah, you are right, this part left from initial implementation